tests/test_arch_soot.py

```diff
@@ -0,0 +1,430 @@
+from archinfo.arch import Endness
+from nose.tools import raises
+from archinfo.arch_soot import SootAddressDescriptor, SootMethodDescriptor, SootAddressTerminator, SootFieldDescriptor, SootClassDescriptor, SootNullConstant, SootArgument, ArchSoot
+import nose.tools
```

Comment:
These should probably be reordered. https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement might be helpful.

Reply:
Okay. I'll do that.